test/e2e/density.go

```diff
@@ -166,6 +176,152 @@ func logPodStartupStatus(c *client.Client, expectedPods int, ns string, observed
 	}
 }
 
+func runDensityTest(dtc DensityTestConfig, startupTime *time.Duration) {
```

Comment:
The function determines the startupTime which is used in AfterEach..  It is probably better to have that as a return from the function though.